xgi/convert.py

```diff
@@ -302,3 +306,103 @@ def to_incidence_matrix(H, sparse=True, index=False):
     from_incidence_matrix
     """
     return xgi.incidence_matrix(H, sparse=sparse, index=index)
+
+
+def from_bipartite_network(G, create_using=None, dual=False):
+    """
+    Create a Hypergraph from a NetworkX bipartite graph.
+
+    Parameters
+    ----------
+    G : nx.Graph()
+        A networkx bipartite graph. Each node in the graph has a property
+        'bipartite' taking the value of 0 or 1 indicating the type of node.
+
+    create_using : Hypergraph constructor, optional
+        The hypergraph object to add the data to, by default None
+    
+    dual : bool, default : False
+        Whether to get edges from bipartite=0 and nodes from bipartite=1
+    
+    Returns
+    -------
+    xgi.Hypergraph
+        The equivalent hypergraph
+
+    Notes
+    -----
+    A partition for the nodes in a bipartite graph generates a hypergraph.
+        >>> import networkx as nx
+        >>> import xgi
+        >>> G = nx.Graph()
+        >>> G.add_nodes_from([1, 2, 3, 4], bipartite=0)
+        >>> G.add_nodes_from(['a', 'b', 'c'], bipartite=1)
+        >>> G.add_edges_from([(1, 'a'), (1, 'b'), (2, 'b'), (2, 'c'), (3, 'c'), (4, 'a')])
+        >>> H = xgi.from_bipartite_network(G)
+    """
+    # TODO: Add filepath keyword to signatures here and with dataframe and numpy array
```

Comment:
Rather than leave TODO comments on the codebase, I'd rather we left issues on GH.